src/bptl/work_units/xential/serializers.py

```diff
@@ -0,0 +1,28 @@
+import binascii
+from base64 import b64decode
+
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+
+class CallbackDataSerializer(serializers.Serializer):
+    bptl_ticket_uuid = serializers.UUIDField(label=_("BPTL ticket UUID"))
+    document = serializers.CharField(help_text=_("Base 64 encoded document content."))
```

Comment:
Should we use https://github.com/Hipo/drf-extra-fields#base64filefield instead?

Reply:
I added this in: https://github.com/GemeenteUtrecht/bptl/pull/126